db.php

```diff
@@ -1,12 +1,6 @@
 <?php
+	include_once('db_creds.php');
```

Comment:
require_once would be better as it will throw an error if the file is not found. Also, something I found out recently is that parentheses are not needed on includes (similar to how return statements in C++ don't need them but Dr. Gibbons always included them).